CRM/Utils/Token.php

```diff
@@ -1829,7 +1829,7 @@ public static function getCustomFieldTokens($entity, $usedForTokenWidget = FALSE
     $customTokens = [];
     $tokenName = $usedForTokenWidget ? "{contribution.custom_%d}" : "custom_%d";
     foreach (CRM_Core_BAO_CustomField::getFields($entity) as $id => $info) {
-      $customTokens[sprintf($tokenName, $id)] = $info['label'];
+      $customTokens[sprintf($tokenName, $id)] = $info['label'] . ' :: ' . $info['groupTitle'];
```

Comment:
The other recent PRs about custom field choices were putting grouptitle first.

Reply:
@demeritcowboy - it looks like 'second' is consistent with other token places - eg. https://github.com/civicrm/civicrm-core/pull/21429/files#diff-c3c60600f080b49f2a9029b82865098c2a2742fbcedce0ecd731376917d80e1bR495 reflects 'current behaviour'

I'm happy to look at flipping but I think I'd rather do it in a follow up if possible because I think there might be a couple of moving parts here